src/scripts.js

```diff
@@ -16,9 +19,27 @@ const favoriteRecipesSection = document.getElementById('favoriteRecipesSection')
 
 const weeklyRecipesSection = document.getElementById('weeklyRecipesSection');
 
+let searchBar = document.querySelector('.favorites-view__favorites');
+let mainGlide = document.querySelector('.search-ingredient-glide__images-container');
 
 homeViewButton.addEventListener('click', displayHomeView);
 allRecipesButton.addEventListener('click', displayAllRecipes);
+searchBar.addEventListener('keyup', (e) => {
```

Comment:
Everything here should be broken out into a separate function versus inline with the event listener.

Also, we have a rule on our team that variable names should be at least three characters long, so we never use `e` and just spell out `event`. This is more preference than anything, but, as you start to work in the field, it's helpful to have _more_ context than less.